logical_switch_port_test.go

```diff
@@ -149,11 +149,11 @@ func TestLogicalSwitchPortAPI(t *testing.T) {
 
 	//validate that setting fields on an empty LSP string gives a nil cmd and an error
 	cmd, err = ovndbapi.LSPSetDynamicAddresses(PORT_TEST_LSP3, PORT_TEST_LSP1DYNADDR1)
-	assert.Equal(t, cmd, nil)
+	assert.Equal(t, cmd == nil, true)
 	assert.Equal(t, err == nil, false)
 
 	cmd, err = ovndbapi.LSPSetExternalIds(PORT_TEST_LSP3, extIds)
-	assert.Equal(t, cmd, nil)
+	assert.Equal(t, cmd == nil, true)
```

Comment:
Thanks @abhat for addressing all the comments and I think I am on same page for sets.
Considering that, I have one final comment to cover the full row update scenario. Can you also cover the same in the UTs by setting one more foo:bar value in externalids for PORT_TEST_LSP1 and make sure getextid is returning multiple keys and not just PORT_TEST_EXT_ID_MAC_KEY after https://github.com/eBay/go-ovn/blob/master/logical_switch_port_test.go#L148. Same for setOptions too to make sure update is including old+new key and not excluding old key.

Reply:
Done.